script.js

```diff
@@ -1 +1,25 @@
-console.log('this works');
\ No newline at end of file
+// Set 1
+
+var div;
+
+// create a buildBox function
+function buildBox() {
+	// create and insert an empty div on the page with: positioning, top and left coordinates, background color expressed as rgba 
+	div = document.createElement('div');
+	div.style.cssText = 'width: 300px; height: 300px; position: absolute; top: 0; left: 0; background-color: rgba(0,0,0,1);';
```

Comment:
A few notes on this.
- setting this as a long string is unwieldy, prone to errors, and difficult to read. Use properties of the style object, as you did below in setStyles
- once we have setStyles, we might as well use it! No need to set the width, height, etc at this point, just set the position: absolute and then run setStyles on it.
- calling setStyles from here is likely to have some issues, as written. Take a look at https://javascriptweblog.wordpress.com/2010/07/06/function-declarations-vs-function-expressions/ and see if you get why that would be.